proxy/common.h

```diff
@@ -134,6 +134,7 @@ extern gchar **srvtypes;
 extern struct namespace_info_s nsinfo;
 gboolean validate_namespace (const char * ns);
 gboolean validate_srvtype (const char * n);
+GError * validate_srvtype_v2 (const char * n);
```

Comment:
You should add a small explanation between validate_srvtype and validate_srvtype_v2